examples/ex08_materials/include/kernels/ExampleDiffusion.h

```diff
@@ -42,6 +42,6 @@ class ExampleDiffusion : public Diffusion
    * This MooseArray will hold the reference we need to our
```

Comment:
I've seen this comment in some app today and was puzzled because users are not required to know that a mat prop is a MooseArray behind the scenes. Looks like a remnant of a distant past.